modules/qualify/qualify.c

```diff
@@ -0,0 +1,300 @@
+/**
+ * @file qualify.c Pinging of peer in CALL_STATE_INCOMING via SIP OPTIONS
+ *
+ * Copyright (C) 2023 Commend.com - m.fridrich@commend.com
+ */
+
+#include <string.h>
+#include <stdlib.h>
+#include <re.h>
+#include <baresip.h>
+
+
+/**
+ * @defgroup qualify module
+ *
+ * This module implements a pinging mechanism using SIP OPTIONS to qualify the
+ * peer while a call is in the INCOMING state to ensure that the peer is
+ * reachable and we will be able to successfully answer the call.
+ *
+ * Configuration:
+ * qual_freq     [seconds]    qualify frequency
+ * qual_to       [ms]         qualify timeout
+ *
+ * The OPTIONS are only sent if both options are present, both are not zero,
+ * qualify_freq is greater than qual_to, and the call is incoming. As soon as
+ * the call is established or closed, we stop sending OPTIONS.
+ * If we run don’t receive a response to an OPTIONS request within the
```

Comment:
- correct this sentence
- Avoid "we"! Use the local UA, or "is stopped", "is sent"!

Reply:
Done.